Source/Config/Main.lua

```diff
@@ -0,0 +1,25 @@
+local DEFAULTS = {
+  ["debug"] = false,
+  ["mailbox tooltips"] = true
+}
+function Auctionator.Config.Set(name, value)
+  if AUCTIONATOR_CONFIG == nil then
+    error("AUCTIONATOR_CONFIG not initialized")
+  else
+    AUCTIONATOR_CONFIG[name] = value
+  end
+end
+
+function Auctionator.Config.Reset()
+  AUCTIONATOR_CONFIG = {}
+end
+
+function Auctionator.Config.Get(name)
+  if AUCTIONATOR_CONFIG == nil then
```

Comment:
Don't have a strong opinion, but do we want to check for existence of the config property (i.e. `Auctionator.Config.Values` includes `name`) to provide a meaningful error message if we screw up a `name` value?